controller/server/view/network_list_page.ml

```diff
@@ -74,7 +75,14 @@ let html { proxy; is_internet_connected; services } =
           ; if List.length services = 0 then
               txt "No services available"
             else
-              ul ~a:[ a_class [ "d-NetworkList" ] ] (List.map service services)
+              ul
+                ~a:[ a_class [ "d-NetworkList" ]; a_role [ "list" ] ]
+                (List.map service services)
+          ]
+
+      ; section
+          [ h2 ~a:[ a_class [ "d-Subtitle" ] ] [ txt "Network Interfaces" ]
```

Comment:
Proposing to label it _Interfaces_, because we’re already in the context of _Network_:

```suggestion
          [ h2 ~a:[ a_class [ "d-Subtitle" ] ] [ txt "Interfaces" ]
```

Reply:
I added the "Network" part on purpose, because to most non-technical users "Interfaces" may not mean anything. I aim to increase the likelihood that when non-technical staff is asked by their sysadmin to provide a MAC address they will find it.

I thought about using "devices" instead of "interfaces", but this is incorrect, as this is a list of interfaces. My hope was that "network interfaces", while somewhat redundant, would reassure the non-technical user that "yes yes, this is the network stuff".